deb-package/deb-build/DEBIAN/control

```diff
@@ -0,0 +1,8 @@
+Package: cryptext
+Version: 1.0-1
```

Comment:
I would not choose the `1.0` version as the development of core features is not finished in my opinion, more `0.1` maybe ;)